TeachingRecordSystem/src/TeachingRecordSystem.Api/V3/Handlers/GetTeacherHandler.cs

```diff
@@ -331,22 +333,32 @@ public class GetTeacherHandler(
             } :
             null;
 
-    private static GetTeacherResponseInduction? MapInduction(dfeta_induction induction, dfeta_inductionperiod[] inductionperiods, AccessMode accessMode) =>
+    private static GetTeacherResponseInduction? MapInduction(dfeta_induction induction, dfeta_inductionperiod[] inductionperiods, AccessMode accessMode, Contact contact) =>
         induction != null ?
             new GetTeacherResponseInduction()
             {
                 StartDate = induction.dfeta_StartDate.ToDateOnlyWithDqtBstFix(isLocalTime: true),
                 EndDate = induction.dfeta_CompletionDate.ToDateOnlyWithDqtBstFix(isLocalTime: true),
-                Status = induction.dfeta_InductionStatus?.ConvertToInductionStatus(),
-                StatusDescription = induction.dfeta_InductionStatus?.GetDescription(),
+                Status = contact.dfeta_InductionStatus?.ConvertToInductionStatus(),
+                StatusDescription = contact.dfeta_InductionStatus?.GetDescription(),
                 CertificateUrl =
-                    (induction.dfeta_InductionStatus == dfeta_InductionStatus.Pass || induction.dfeta_InductionStatus == dfeta_InductionStatus.PassedinWales) &&
+                    (contact.dfeta_InductionStatus == dfeta_InductionStatus.Pass || contact.dfeta_InductionStatus == dfeta_InductionStatus.PassedinWales) &&
                         induction.dfeta_CompletionDate is not null &&
                         accessMode == AccessMode.IdentityAccessToken ?
                     "/v3/certificates/induction" :
                     null,
                 Periods = inductionperiods.Select(MapInductionPeriod).ToArray()
             } :
+            contact.dfeta_qtlsdate.HasValue ?
+                    new GetTeacherResponseInduction()
+                    {
+                        StartDate = null,
+                        EndDate = null,
+                        Status = contact.dfeta_InductionStatus?.ConvertToInductionStatus(),
```

Comment:
As above, there shouldn't be an induction object returned at all if there's no status